app/components/trainee_status_card/view.rb

```diff
@@ -2,34 +2,28 @@
 
 module TraineeStatusCard
   class View < GovukComponent::Base
-    attr_reader :state, :count, :target
+    attr_reader :status, :count, :target
 
     STATUS_COLOURS = {
-      draft: "grey",
```

Comment:
are we removing all these different statuses completely? My understanding is this is the tag on the trainee card itself (might have got that wrong)

I guess i'm wondering if we definitely don't want to show some of these labels anymore i.e. withdrawn

Reply:
It's actually the homepage square. If you go to /view_components and find the component you'll see:

<img width="906" alt="Screenshot 2022-06-27 at 11 04 16" src="https://user-images.githubusercontent.com/18436946/175916977-74c62b85-1132-422a-b4b5-44c577277e3a.png">

You might be thinking of `StatusTag` which is:

<img width="685" alt="Screenshot 2022-06-27 at 11 06 08" src="https://user-images.githubusercontent.com/18436946/175917221-d3328797-fb94-46e4-8004-70524b011301.png">

These are unchanged, we're still showing them all.

Reply:
Ah amazing! Sorry my mistake :)